src/animation.js

```diff
@@ -141,7 +144,15 @@
     play: function() {
       this._paused = false;
       if (this._isFinished || this._idle) {
-        this._currentTime = this._playbackRate > 0 ? 0 : this._totalDuration;
+        if (this._playbackRate >= 0) {
```

Comment:
It kind of emulates the part in the spec that says "Set animation’s hold time to zero." when we don't really have the concept of a hold time here.